confirmDialog.js

```diff
@@ -4,19 +4,19 @@ const ModalDialog = imports.ui.modalDialog;
 const CheckBox = imports.ui.checkBox;
 const Clutter = imports.gi.Clutter;
 
-const Gettext = imports.gettext.domain("gnome-trash");
+const Gettext = imports.gettext.domain("clipboard-indicator");
 const _ = Gettext.gettext;
 
```

Comment:
You can remove these now. This will make this confirm-dialog class reusable without a change :+1:

Reply:
Unfortunately the translation won't work if I remove it. I don't like it as well.

Reply:
I just update the French translation with these strings, and removed these 2 lines : works like a charm :)

Reply:
How about `Gettext.gettext`? Do you import gettext?

Reply:
I removed both lines. There's no use for "_" in that file.